goose3/extractors/images.py

```diff
@@ -412,3 +414,10 @@ def load_customesite_mapping(self):
         for line in lines:
             domain, css = line.split('^')
             self.custom_site_mapping.update({domain: css})
+    
+    def add_schema_if_none(self, src):
+        src_test = parse_url(src)
```

Comment:
`parse_url` is not imported here

Reply:
@cdrappi why not just import parse_url?

Reply:
which one? i see no `parse_url` used anywhere else in the project

Reply:
good point. I assumed the author knew what they were doing.

Could you decipher what they were trying to do and contribute?

Reply:
@lababidi i believe this is what the author intended to do. https://github.com/goose3/goose3/pull/11

disclosure, i have little skin in the game as my usecase has no need for image stuff